### DISCUSSION TITLE
Which logo do you prefer?

### DISCUSSION THREAD

**Discussion by rustdesk** [rustdesk/rustdesk#3087] [Polls]
![image](https://user-images.githubusercontent.com/71636191/216910157-a0c7ed2d-6678-4152-8648-8d861e8be7a3.png)

It was original designed by @NicKoehler https://github.com/rustdesk/rustdesk/issues/2722#issuecomment-1413541303, and I made a little modification.

**Comment by SZP1206** [comment]
Personally, I prefer the left one.
The left one is more lively, while the right one is more serious. Considering the product positioning, I think the right one is more suitable.

**Reply by cvladan** [reply]
Seriously?

**Comment by ameaninglessname** [comment]
Personally, I prefer the right one.
The right one is more attracting my eyes, while the left one is more good-looking.

**Reply by rustdesk** [reply]
I have chosen the good-looking one, because my daughter prefers it. :)
